modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -206,9 +206,10 @@ sda.enkf.multisite <- function(settings, obs.mean, obs.cov, Q = NULL, restart=F,
     ###  Taking care of Forecast. Splitting /  Writting / running / reading back###
     ###-------------------------------------------------------------------------###-----  
     #- Check to see if this is the first run or not and what inputs needs to be sent to write.ensemble configs
+    
     if (t>1){
       #removing old simulations
-      unlink(list.files(outdir, "*.nc", recursive = T, full.names = T))
+      unlink(list.files(outdir, "*.nc.var", recursive = TRUE, full.names = TRUE))
```

Comment:
why are we just removing old nc.var and not .nc as well?  Previously it looks like we were removing nc data?  Why remove at all?

Reply:
This was for checking the .nc files that get output at the end of each year. The SDA originally deleted all .nc or .nc.var files, so I couldn't check the individual years.

Reply:
@serbinsh This isn't going in to the final SDA, it was just a section I added to dig into issues